recipe/build.sh

```diff
@@ -4,21 +4,33 @@ cd nss
 
 if [[ ${HOST} =~ .*darwin.* ]]; then
     USE_GCC=0
+    MACOS_SDK_DIR="${CONDA_BUILD_SYSROOT}"
 elif [[ ${HOST} =~ .*linux.* ]]; then
     USE_GCC=1
 fi
 
-make    \
-    USE_64=1 \
-    PREFIX="${PREFIX}" \
-    NSPR_PREFIX="${PREFIX}" \
-    NSPR_INCLUDE_DIR="${PREFIX}/include/nspr" \
-    C_INCLUDE_PATH="${PREFIX}/include:${PREFIX}/include/nspr" \
-    LIBRARY_PATH="${PREFIX}/lib" \
+make   -j1 BUILD_OPT=1 \
```

Comment:
I think we can try to use multiple cores here to compile, may we?

Reply:
I didn't add that. :)

Probably, but I would save it for another PR in the interest of making progress on the compiler migration.

Reply:
Ok then! :)